index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FIGMA-1</title>
```

Comment:
- [ ] For accessibility reasons, SEO, and a good user experience, I think you should consider using a descriptive text here instead of this generic Figma because it is not about about Figma. Something like: `Chandu's Portfolio` would do.